src/languages/en.js

```diff
@@ -168,6 +168,7 @@ export default {
         sizeNotMet: 'Attachment size must be greater than 240 bytes.',
         wrongFileType: 'Attachment is the wrong type',
         notAllowedExtension: 'This filetype is not allowed',
+        folderNotAllowedMessage: 'Uploading folder is not allowed.'
```

Comment:
@luacmartins Sorry to bother again.
Whom should we confirm the error message with? In most cases previously internal engineers would provide the required string AFAIK.

Reply:
@luacmartins Gentle ping on this confirmation

Reply:
@abdulrahuman5196 thanks for the ping, I missed the first one 😬 I added the `Waiting for copy` label in the issue to get someone to review the copy. Let's see what they say.

Reply:
@Charan-hs @abdulrahuman5196 the updated copy is: `Uploading a folder is not allowed. Try a different file.`